### ISSUE TITLE
image link from 3-part webside cant showed

### ISSUE THREAD

**Issue by yyzfish** [dragonman225/notablog#35]
if i use medium article picture like below
https://user-images.githubusercontent.com/69407830/159119772-ed49e127-d3d0-4e6c-9845-a8d556ffeb29.png
it can show currect in notion ,but no in notablog html file

**Comment by dragonman225** [comment]
The image shows if I directly visit the URL, so it should work. Can you share the link to the blog post with the broken image?

**Comment by yyzfish** [comment]
yes,when it direct open ,its work ,but when in notablog,it  cant show up

**Comment by dragonman225** [comment]
Could you share the link of your blog? So I can look into what goes wrong.

By the way, if you view the html files locally, some images may not work because the browser blocks them for security reasons.
